plugins/source/awspricing/client/spec.go

```diff
@@ -1,10 +1,15 @@
 package client
 
+import (
+	"github.com/cloudquery/plugin-sdk/v4/scheduler"
+)
+
 type Spec struct {
 	RegionCodes []string `json:"region_codes,omitempty"`
 	OfferCodes  []string `json:"offer_codes,omitempty"`
 
-	Concurrency int `json:"concurrency,omitempty"`
+	Concurrency int                `json:"concurrency,omitempty"`
```

Comment:
IMO this should be a separate struct in the plugin SDK called `SchedulerConfig` & we might need to have some setting like
```yaml
scheduler:
  concurrency: 1000
  strategy: round-robin
```
instead of putting the sched cfg to the spec top level

Reply:
This PR aims to first fix the bug. We can discuss changing the spec (which we'll need to do in a backwards compatible way or as a breaking change) in a separate PR (or even better, an issue). WDYT?

Reply:
100% this should go into a separate discussion, maybe even a separate issue in the plugin-sdk